<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Confirm</class>
 <widget class="QWidget" name="Confirm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="ConfirmWidget" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>741</width>
     <height>621</height>
    </rect>
   </property>
   <widget class="QLabel" name="OrderPlacedLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Order Successfully Placed!</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>601</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>Order Summary</string>
    </property>
    <widget class="QLabel" name="SubTotalLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>subtotal: EGP 20.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="DeliveryLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delivery Fee: EGP 20.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="TotalLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Total: EGP 20.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="DeliveringToLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delivering To:</string>
     </property>
    </widget>
    <widget class="QLabel" name="AddressLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Al Rehab City, Al Rehab City 1, Othman Ibn Affan</string>
     </property>
    </widget>
    <widget class="QLabel" name="PaymentMethodLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Payment Method:</string>
     </property>
    </widget>
    <widget class="QLabel" name="PaymentMethod">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Cash on Delivery (COD)</string>
     </property>
    </widget>
    <widget class="QLabel" name="DelievryDateLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delivery Date:</string>
     </property>
    </widget>
    <widget class="QLabel" name="DeliveryDate">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>300</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>5 buisness Days</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(249, 184, 158);</string>
    </property>
    <property name="text">
     <string>Continue Shopping</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
